### PR TITLE
SAML updates to IDPS page

### PR DESCRIPTION
**Pull request by susanharper-okta**
## Description:
- **What's changed?** SAML updates to IDPS page
- **Is this PR related to a Monolith release?** No

### Resolves:

* [OKTA-332191](https://oktainc.atlassian.net/browse/OKTA-332191)

### CODE REVIEW

**Review by larsjohansen-okta — APPROVED** [review]
To me this looks good, however can you provide screenshots to show how it looks?

**Comment by susanharper-okta** [comment]
@larsjohansen-okta 
Screen Shot 2020-09-25 at 8 57 34 AM
Screen Shot 2020-09-25 at 9 00 48 AM
Screen Shot 2020-09-25 at 8 58 50 AM
Screen Shot 2020-09-25 at 9 02 51 AM

**Review by venkatviswanathan-okta — APPROVED** [review]
lgtm 🚢

File: packages/@okta/vuepress-site/docs/reference/api/idps/index.md [hunk 1]
```diff
@@ -4420,10 +4422,9 @@ Content-Type: application/json
 
 ### Link a User to a social provider without a Transaction
 
-
 <ApiOperation method="POST" url="/api/v1/idps/${idpId}/users/${userId}" />
 
-Links an Okta User to an existing [social provider](#identity-provider-object). This endpoint doesn't support the SAML2 [Identity Provider type](#identity-provider-type).
+Links an Okta User to an existing [social provider](#identity-provider-object). To enable persistent account linking with a SAML Identity Provider, ensure that `honorPersistentNameId` is set to `true` and the [Name Identifier Format](/docs/reference/api/idps/#saml-2-0-settings-object) of the incoming assertion is `urn:oasis:names:tc:SAML:2.0:nameid-format:persistent`.
```
Comment (yuliu-okta) [inline packages/@okta/vuepress-site/docs/reference/api/idps/index.md:4427]: Links an Okta User to an existing SAML or social provider. The SAML Identity Provider must have `honorPersistentNameId` is set to `true` to be able to use this API. The [Name Identifier Format](/docs/reference/api/idps/#saml-2-0-settings-object) of the incoming assertion must be `urn:oasis:names:tc:SAML:2.0:nameid-format:persistent` to honor this link.

File: packages/@okta/vuepress-site/docs/reference/api/idps/index.md [hunk 2]
```diff
@@ -5077,16 +5079,19 @@ Determines the [IdP Key Credential](#identity-provider-key-credential-object) us
...
-| nameFormat | The name identifier format to use | String      | TRUE     | FALSE    | [SAML 2.0 Name Identifier Formats](#saml-2-0-name-identifier-formats) | urn:oasis:names:tc:SAML:1.1:nameid-format:unspecified |
+| Property   | Description                                                                                                  | DataType| Nullable | Readonly | Default                                              |
+| ---------- | ------------------------------------------------------------------------------------------------------------ | ------- | -------- | -------- | ---------------------------------------------------- |
+| nameFormat | The name identifier format to use. See [SAML 2.0 Name Identifier Formats](#saml-2-0-name-identifier-formats) | String  | TRUE     | FALSE    | urn:oasis:names:tc:SAML:1.1:nameid-format:unspecified|
+| honorPersistentNameId | Determines if the IdP should persist account linking when the incoming assertion NameID format is `urn:oasis:names:tc:SAML:2.0:nameid-format:persistent`| Boolean | FALSE | FALSE | FALSE|
```
Comment (yuliu-okta) [inline packages/@okta/vuepress-site/docs/reference/api/idps/index.md:5085]: Is Nullable TRUE or FALSE?

Comment (venkatviswanathan-okta) [reply packages/@okta/vuepress-site/docs/reference/api/idps/index.md:5085]: FALSE right?

Comment (yuliu-okta) [reply packages/@okta/vuepress-site/docs/reference/api/idps/index.md:5085]: Just tested, it is Nullable, it is default to FALSE

**Review by yuliu-okta — APPROVED** [review]

**Pull request merged**